core/src/test/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/UrlHelperTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.pierfrancescosoffritti.androidyoutubeplayer.core
+
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.utils.getVideoIdFromUrl
+import org.junit.Test
+
+class UrlHelperTest {
+
+    @Test
+    fun testGetVideoIdFromUrl_WithUrlHasIdAndUrlSeparator_ReturnsVideoId(){
```

Comment:
Remove all the methods, write a single method called `testGetVideoIdFromUrl`

Reply:
we should have a separate test method for each scenario, or do you mean another thing ?

Reply:
We can have a single function testing multiple cases. These different functions can be merged into one, with all the tests in it.

Reply:
yes we can have, but we should not each test case should be have a single test/assert/logic to check. 

also, what impact of having them **separated** (which make code more readable, and we can get the business from name of test cases) ?